docs/source/tutorials/running_a_simulation/interactive.rst

```diff
@@ -265,7 +280,12 @@ population size to be smaller so the simulation takes less time to run.
 
 .. code-block:: python
 
-   sim.configuration.update({'population': {'population_size': 1_000}})
+    # Setting attributes ensures you are updating existing keys, rather than
+    # creating new ones
+    sim.configuration.population.population_size = 1_000
```

Comment:
does this still work with the latest config tree?

Reply:
I know it runs, because of the doctest below. However, I didn't test that it actually worked in the latest version of Vivarium. I see no concerning changes to [the relevant code](https://github.com/ihmeuw/vivarium/blob/710b7344661257a898cbaf80b20cee39569bfffe/src/vivarium/config_tree.py#L514-L521), though.

Reply:
Ah good point -- I guess I wasn't very clear here. This is intended to show two options, where you would only do one of them. I could remove the `update` option, or comment it out.